tensorrt_llm/commands/serve.py

```
@@ -1682,6 +1866,270 @@ def disaggregated(
+
+    public_host, public_port = disagg_cfg.hostname, disagg_cfg.port
+    coord_port = int(
+        os.environ.get(DisaggWorkerEnvs.TLLM_DISAGG_COORDINATOR_PORT,
+                       public_port - 1))
+    # The fleet is co-located with the implicit coordinator, so route its hot
+    # /select,/finish over a Unix domain socket (avoids the TCP loopback stack
+    # that dominated per-request latency). Opt-out via the UDS env = "0".
+    use_uds = os.environ.get(DisaggWorkerEnvs.TLLM_DISAGG_COORDINATOR_UDS,
+                             "1") == "1"
+    coord_uds = (
+        f"/tmp/trtllm_disagg_coord_{public_port}.sock"  # nosec B108
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/trtllm_disagg_coord_"

(S108)